app/register/index.js

```diff
@@ -0,0 +1,57 @@
+const validatePassword = () => {
+	const passwordElement = document.getElementById('password');
+	const submitButton = document.getElementById('submit-button');
+
+	const hideWarningMessageAndDisableSubmit = (warningMessageElement) => {
+		document.getElementById(warningMessageElement).classList.toggle('hidden');
+		submitButton.disabled = true;
+	};
+
+	const checkPassword = (event) => {
+		if (event.target.value.length < 8) {
+			hideWarningMessageAndDisableSubmit('length');
+		}
+
+		const regExCheckNumber = /([0-9])/;
+		if (!regExCheckNumber.test(event.target.value)) {
+			hideWarningMessageAndDisableSubmit('number');
+		}
+
+		const regExCheckLowercase = /([a-z])/;
+		if (!regExCheckLowercase.test(event.target.value)) {
+			hideWarningMessageAndDisableSubmit('lowercase');
+		}
+
+		const regExCheckUppercase = /([A-Z])/;
+		if (!regExCheckUppercase.test(event.target.value)) {
+			hideWarningMessageAndDisableSubmit('uppercase');
+		}
+	};
+
+	const removeWarningMessages = () => {
+		const warningMessages = document
```

Comment:
I think this could be refactored in a future PR so that I could pass in an id for any div containing warning messages (such as the email div that will be in the next PR) and add the hidden class to all of it's p tags.